crates/swc_xml_parser/src/parser/open_elements_stack.rs

```diff
@@ -0,0 +1,31 @@
+use crate::parser::RcNode;
+
+pub struct OpenElementsStack {
+    pub items: Vec<RcNode>,
```

Comment:
@kdy1 I just remove `pop` and `push`, because they are just run `pop` and `push` on `items`, removing `pub` is not possible, because we use `last` and more methods on `items`